x-pack/elastic-agent/_meta/elastic-agent.yml

```diff
@@ -9,6 +9,26 @@ outputs:
     password: changeme
 
 inputs:
+  - type: synthetics/monitors
+    streams:
+      - type: http
+        name: MyHTTPMonitor
+        id: my-id
+        target: "https://elastic.co"
```

Comment:
Should target be an array perhaps to simplify manual configuration? Conflicts with id I assume.